java/src/main/native/src/TableJni.cpp

```diff
@@ -159,81 +154,84 @@ class jni_writer_data_sink final : public cudf::io::data_sink {
     current_buffer_written = 0;
   }
 
-  void handle_buffer(JNIEnv* env, jobject buffer, jlong len) {
+  void handle_buffer(JNIEnv *env, jobject buffer, jlong len) {
     env->CallVoidMethod(callback, handle_buffer_method, buffer, len);
     if (env->ExceptionCheck()) {
       throw std::runtime_error("handleBuffer threw an exception");
     }
   }
 
-  JavaVM* jvm;
+  JavaVM *jvm;
   jobject callback;
   jmethodID handle_buffer_method;
   jobject current_buffer = nullptr;
-  char * current_buffer_data = nullptr;
+  char *current_buffer_data = nullptr;
   long current_buffer_len = 0;
   long current_buffer_written = 0;
   size_t total_written = 0;
   long alloc_size = MINIMUM_WRITE_BUFFER_SIZE;
 };
 
-template <typename STATE>
-class jni_table_writer_handle final {
+template <typename STATE> class jni_table_writer_handle final {
 public:
-  explicit jni_table_writer_handle(std::shared_ptr<STATE> & state) : state(state), sink() {}
-  jni_table_writer_handle(std::shared_ptr<STATE> & state,
-          std::unique_ptr<jni_writer_data_sink> & sink) : state(state), sink(std::move(sink)) {}
+  explicit jni_table_writer_handle(std::shared_ptr<STATE> &state) : state(state), sink() {}
+  jni_table_writer_handle(std::shared_ptr<STATE> &state,
+                          std::unique_ptr<jni_writer_data_sink> &sink)
+      : state(state), sink(std::move(sink)) {}
 
   std::shared_ptr<STATE> state;
   std::unique_ptr<jni_writer_data_sink> sink;
 };
 
-typedef jni_table_writer_handle<cudf::experimental::io::detail::parquet::pq_chunked_state> native_parquet_writer_handle;
-typedef jni_table_writer_handle<cudf::experimental::io::detail::orc::orc_chunked_state> native_orc_writer_handle;
+typedef jni_table_writer_handle<cudf::experimental::io::detail::parquet::pq_chunked_state>
+    native_parquet_writer_handle;
+typedef jni_table_writer_handle<cudf::experimental::io::detail::orc::orc_chunked_state>
+    native_orc_writer_handle;
 
 /**
- * Take a table returned by some operation and turn it into an array of column* so we can track them ourselves
- * in java instead of having their life tied to the table.
+ * Take a table returned by some operation and turn it into an array of column* so we can track them
+ * ourselves in java instead of having their life tied to the table.
  * @param table_result the table to convert for return
  * @param extra_columns columns not in the table that will be added to the result at the end.
  */
-static jlongArray convert_table_for_return(JNIEnv * env,
-        std::unique_ptr<cudf::experimental::table> &table_result,
```

Comment:
There are a lot of formatting changes in here that make it difficult to see what functionality has changed and what has not change